scipy/stats/_stats_py.py

```diff
@@ -671,16 +679,19 @@ def tvar(a, limits=None, inclusive=(True, True), axis=0, ddof=1):
     20.0
 
     """
-    a = asarray(a)
-    a = a.astype(float)
+    if not np.issubdtype(a.dtype, np.inexact):
+        a = a.astype(np.float64)
     if limits is None:
         return a.var(ddof=ddof, axis=axis)
     am = _mask_to_limits(a, limits, inclusive)
-    amnan = am.filled(fill_value=np.nan)
+    amnan = am.filled(fill_value=_get_nan(a))
     return np.nanvar(amnan, ddof=ddof, axis=axis)
 
 
-def tmin(a, lowerlimit=None, axis=0, inclusive=True, nan_policy='propagate'):
+@_axis_nan_policy_factory(
+    lambda x: x, n_outputs=1, result_to_tuple=lambda x: (x,)
+)
+def tmin(a, lowerlimit=None, axis=0, inclusive=True):
```

Comment:
Something I didn't think about - if you remove the argument `nan_policy`, I don't think it will be positional anymore, just keyword.

So I don't think I've removed it in the past. You could remove it's documentation if you want. Maybe add comments that it is handled by the decorator. (I've done that inconsistently.)

Reply:
done.